sceptre/synapsedev/templates/accounts.yaml

```diff
@@ -71,10 +71,10 @@ Resources:
       LoginProfile:
         Password: !Ref InitNewUserPassword
         PasswordResetRequired: true
-  AWSIAMPeterHarveyUser:
+  AWSIAMLeonLiUser:
     Type: 'AWS::IAM::User'
     Properties:
-      UserName: peter.harvey@sagebase.org
+      UserName: leon.li@sagebase.org
```

Comment:
We are actively denying creation of user LoginProfile at the org root level https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/organization.yaml#L15